js_modules/dagit/packages/ui/src/components/BaseTag.tsx

```diff
@@ -25,7 +38,18 @@ export const BaseTag = (props: Props) => {
   return (
     <StyledTag $fillColor={fillColor} $interactive={interactive} $textColor={textColor}>
       {icon || null}
-      {label !== undefined && label !== null ? <span>{label}</span> : null}
+      {label !== undefined && label !== null ? (
+        <span
+          data-tooltip={label}
```

Comment:
This causes problems when `label` is a React node.

<img width="162" alt="image" src="https://user-images.githubusercontent.com/2823852/157542349-17a02aa8-48b3-497d-ae67-980331a311c8.png">

Hover:

<img width="173" alt="Screen Shot 2022-03-09 at 1 51 05 PM" src="https://user-images.githubusercontent.com/2823852/157542366-b7eb2577-3633-4dd1-92eb-b3b4bef5bc05.png">

I guess we could allow specifying a text-only label prop that would appear here, though there might be other good ways to do this too.